x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/MultiClusterSpecIT.java

```diff
@@ -50,8 +59,27 @@ public class MultiClusterSpecIT extends EsqlSpecTestCase {
     @ClassRule
     public static TestRule clusterRule = RuleChain.outerRule(remoteCluster).around(localCluster);
 
-    public MultiClusterSpecIT(String fileName, String groupName, String testName, Integer lineNumber, CsvSpecReader.CsvTestCase testCase) {
-        super(fileName, groupName, testName, lineNumber, convertToRemoteIndices(testCase));
+    @ParametersFactory(argumentFormatting = "%2$s.%3$s")
+    public static List<Object[]> readScriptSpec() throws Exception {
```

Comment:
Can we use the ParametersFactory of the super class?

Reply:
I just copied this here - so as to exclude the async test scenarios.  Eventually this should be removed.